README.md

```diff
@@ -79,6 +81,22 @@ Install CSI Driver:
   * kubectl create -f deploy/kubernetes/csi-snapshotter
   * https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes/csi-snapshotter
 
+### Validating Webhook
```

Comment:
@xing-yang @yuxiangqian let's look at this before we release, but I think it would be good to revamp the overview to make it more clear of all the pieces in this repo and who is responsible for what:

- CRDs + webhook
- controller
- sidecar
- client apis

Reply:
Sure.